AppWidget/glance-widget/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+  ~ Copyright (C) 2021 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<resources>
+    <!-- These attributes are only used for the preview layouts -->
+    <declare-styleable name="AppWidgetAttrs">
```

Comment:
As we need these resolved as resources anyway for Glance, I suggest to define them as resources instead of attributes. A theme is good to allow other apps to re-define theme .. here we're defining the final app, so there isn't much point really.

Reply:
Sounds good, done.